hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/throttles/LeakyBucketDeterministicThrottle.java

```diff
@@ -54,6 +54,27 @@ public boolean allow(@NonNull final Instant now, final long throttleLimit) {
         return delegate.allow(throttleLimit, elapsedNanos);
     }
 
+    /**
+     * Calculates the amount of nanoseconds that elapsed since the last time the bucket was updated
+     * and then consumes the specified amount of capacity.
+     * If there isn't enough capacity, consumes whatever capacity is available and returns (does not throw).
+     *
+     * @param now           - the instant against which the {@link LeakyBucketThrottle} is evaluated.
+     * @param capacityToUse - the amount of capacity to consume
+     */
+    public void useCapacity(@NonNull final Instant now, final long capacityToUse) {
```

Comment:
how is this method and the delegated one meaningfully different than the `allow` method?